src/pages/BanquetFacility/BanquetFacility.js

```diff
@@ -16,12 +21,17 @@ const BanquetFacility = ({data}) => {
     <Layout>
       <Container>
       <div className="banquet-facility">
+        <div className="banquet-facility__triple">
+          <Image className="banquet-facility__image--triple" img={bfimg1} alt=""></Image>
+          <Image className="banquet-facility__image--triple" img={bfimg4} alt=""></Image>
+          <Image className="banquet-facility__image--triple" img={bfimg2} alt=""></Image>
```

Comment:
for now we should be doing something like (that link doesnt exist right now (you'll want to upload the image to wordpress) 

```
<Image className="banquet-facility__image--triple" img="https://dev.huoa.org/banquet-facility-image-1.jpg" alt=""></Image>
```